services/horizon/internal/actions_metrics.go

```diff
@@ -1,9 +1,9 @@
 package horizon
 
 import (
-	metrics "github.com/rcrowley/go-metrics"
 	"github.com/kinecosystem/go/services/horizon/internal/actions"
 	"github.com/kinecosystem/go/support/render/hal"
+	metrics "github.com/rcrowley/go-metrics"
```

Comment:
i think we should avoid invariant changes like sorting import lines, in order to avoid painful merge conflicts when fetching future versions from stellar/go (see https://github.com/kinecosystem/go/pull/32/commits/ef96ac282367068ebd1e925071e173f126563e7b , https://github.com/kinecosystem/go/pull/32/commits/3b8e6a0878d5d5d50817b5c1bac0d3749fcd672c and https://github.com/kinecosystem/go/pull/32/commits/9e750d2847279b7298ff58fad9752afc002001d5 as an example , taken from https://github.com/kinecosystem/go/pull/32 . this was done manually by hand and took quite some time to finish)

i guess GoLand does this by running `gofmt` automatically behind the scenes.

Reply:
I really hope that Stellar is using the standard convention, and for best practice, we should follow the same guidelines.
btw, the PR you mentioned looks like a total mess, to resolve the conflicts I probably just add only the new imports to our version and let the IDE (gofmt or idk what's running in the background) do the formatting.
Sure it may take a couple of minutes to do it but it keeps in the same standards